onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3572,17 +3572,87 @@ def aten_max_pool1d_with_indices(
     raise NotImplementedError()
 
 
+@torch_op("aten::max_pool2d", trace_only=True)
 def aten_max_pool2d(
     self: TensorType,
```

Comment:
Should be: tensor(float16), tensor(float), tensor(double), tensor(int8), tensor(uint8)